PayPalCheckoutWeb/src/main/java/com/paypal/android/checkoutweb/PayPalCheckoutListener.kt

```diff
@@ -0,0 +1,26 @@
+package com.paypal.android.checkoutweb
+
+import com.paypal.android.core.PayPalSDKError
+
+/**
+ * Implement this callback to receive results from [PayPalWebClient].
+ */
+interface PayPalCheckoutListener {
```

Comment:
For consistency can we prefix all classes in this module with `PayPalCheckoutWeb` and in the other module prefix everything with `PayPalCheckout`?

Reply:
And methods too I suppose

Reply:
Or actually I guess the naming convention decided was ` PayPalWebCheckout` and `PayPalNativeCheckout` so let's use those for the module names and class name prefixes

Reply:
Would we ever share some of these interface methods between web and native flows?

Reply:
Hmmm possibly I'm not sure that would work with the idea of these being independent modules